README.md

```diff
@@ -1 +1,50 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+In this lesson we're going to be looking into AngularJS (commonly referred to as just "Angular"), an open-source framework that was released in 2009. It is currently being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
```

Comment:
remove this line, since we don't go into what MVVM and business logic are in this README. Instead, just transition into the rest of the lesson, i.e. "We'll talk about front end frameworks, the problems they solve, and how Angular can help you be a better developer" or something along those lines.